src/components/ContentUploader/ContentUploader.js

```diff
@@ -66,6 +66,7 @@ type Props = {
     onComplete: Function,
     onError: Function,
     onMinimize?: Function,
+    onStage: Function,
     onUpload: Function,
     onUpload: Function,
```

Comment:
Can you remove one of these, looks like there are two

Reply:
There is only one. There are two onUploads.